INSTALL.md

```diff
@@ -42,8 +42,9 @@ cargo install alacritty
         12. [Gentoo](#gentoo)
         13. [Clear Linux](#clear-linux)
         14. [GNU Guix](#gnu-guix)
-        15. [Windows](#windows)
-        16. [Other](#other)
+        15. [Alpine Linux](#alpine-linux)
```

Comment:
You've put it before GNU Guix in the actual docs, but it's after Guix here. So please change one of them.

Reply:
Fixed!